### PR TITLE
Editable columns fix & demo [issue-35]

### PR DESCRIPTION
**Pull request by JaySunSyn**
See https://github.com/Neovici/cosmoz-omnitable/issues/35

- Columns now fire an event if items change and editable-demo shows the change.
- Fixed editing in some columns.

### CODE REVIEW

File: demo/editable-demo.html [hunk 1]
```diff
@@ -55,13 +77,14 @@
...
-					<cosmoz-omnitable-column-time title="Time" value-path="time" editable></cosmoz-omnitable-column-time>
-					<cosmoz-omnitable-column title="Sub-property" value-path="sub.subProp" editable></cosmoz-omnitable-column>
-					<cosmoz-omnitable-column title="Random String" value-path="randomString" editable></cosmoz-omnitable-column>
+					<cosmoz-omnitable-column-amount title="Amount" name="amount" value-path="amount" editable></cosmoz-omnitable-column-amount>
+					<cosmoz-omnitable-column-date title="Date" name="date" value-path="date" editable></cosmoz-omnitable-column-date>
+					<cosmoz-omnitable-column-datetime title="Datetime" name="datetime" value-path="date" editable></cosmoz-omnitable-column-datetime>
+					<cosmoz-omnitable-column-time title="Time" value-path="date" editable></cosmoz-omnitable-column-time>					
+					<cosmoz-omnitable-column title="Sub-property" name="subproperty" value-path="sub.subProp" editable></cosmoz-omnitable-column>
+					<cosmoz-omnitable-column title="Random String" name="randomstring" value-path="randomString" editable></cosmoz-omnitable-column>
```
Comment (plequang) [inline demo/editable-demo.html]: Several columns are missing name attribute

File: demo/editable-demo.html [hunk 2]
```diff
@@ -55,13 +77,14 @@
...
 					<!-- sample column that overrides edit-cell -->
-					<cosmoz-omnitable-column title="Name" name="name" editable>
+					<cosmoz-omnitable-column id="nameCol" title="Name" name="name" value-path="name" editable>
```
Comment (plequang) [inline demo/editable-demo.html]: The name column misses a `value-path` attribute

Comment (JaySunSyn) [reply demo/editable-demo.html]: I think I added it here..

**Comment by JaySunSyn** [comment]
@plequang are you sure you didn't test the old version? Do you see this?

image

**Comment by plequang** [comment]
OK my bad. I don't know what I did wrong when fetching the PR.
Can see the value-path and the info box now.

**Pull request merged**